task/README.md

```diff
@@ -0,0 +1,17 @@
+# Example Task
+
+This is an example Lambda Java task that uses the Cumulus Message Adapter. 
+
+The business logic for the task is in `TaskLogic.java`. If the `TOPIC_ARN` environment variable is configured, a sample message is published to SNS. Sample JSON is returned by the function and will be included in the message adapter output. The business logic function also demonstrates using the `AdapterLogger` for logging.
+
+## Lambda Configuration
+
+The handler should be set to `test_task.task.Task::handleRequest`.
+
+## SNS Configuration
+
+To use this to send a message to SNS, first a topic must be created. 
+
+Configure the topic to publish to by setting an environment variable `TOPIC_ARN` to the topic Arn.
```

Comment:
question? could we send the topic_arn as a config property via the event object?

We have been trying this general approach to task writing (this is not documented anywhere btw) that encourages tasks to receive all the info they need via the `event` input. Setting an Env variable should happen during deployment, which makes it more complicated and more difficult to test.

What do you think?